engine/src/main/resources/org/camunda/bpm/engine/impl/mapping/entity/Authorization.xml

```diff
@@ -55,11 +59,20 @@
       USER_ID_ = #{userId, jdbcType=VARCHAR},
       RESOURCE_TYPE_ = #{resourceType, jdbcType=INTEGER},
       RESOURCE_ID_ = #{resourceId, jdbcType=VARCHAR},
-      PERMS_ = #{permissions, jdbcType=INTEGER}
+      PERMS_ = #{permissions, jdbcType=INTEGER},
+      REMOVAL_TIME_ = #{removalTime, jdbcType=TIMESTAMP}
```

Comment:
This is a bit inconsistent with removal time not being part of the authorization's persistent state (`#getPersistentState`). I don't see a concrete problem here and believe both of the following should work:

* Remove `REMOVAL_TIME_` from the update mapping: With removal time strategy start, we always set the removal time when the permission is inserted and it can never change later
* Add removalTime to the persistent state. This makes it consistent, although probably not strictly necessary as the removal time is never updated on its own. Could avoid future bugs though.

Reply:
Respecting the case in my comment before [1], I would prefer the second option.

[1] https://github.com/camunda/camunda-bpm-platform/pull/740#discussion_r399154251

Reply:
I would still prefer the second option since we need to update the authorization in CAM-11617.